### DISCUSSION TITLE
[How to] bypass windows antyvirus alerts

### DISCUSSION THREAD

**Discussion by gucio321** [AllenDang/giu#961] [Q&A]
Hi there!
If you've ever tried to run giu apps on windows 10, you had to notice that Windows Deffender's alerts have been triggered.
Is there any known way to make the executable "trusted"?

**Comment by Rusty-Weasel** [comment]
Either Windows Defender or ESET Smart Security make any alerts.

Do you have something obvious functions build in your source code? that the alert occurs.

**Comment by gucio321** [comment]
hmm, as far as I remember I was getting feedback from my friends who I shown my builds (https://github.com/gucio321/tic-tac-go as well as example/gifdecode) about antyvirus alerts (it was iirc windows defender _or_ warning about "malicious softwere" while running exe). I've compiled my apps using my linux machine if it matters here.

**Comment by Rusty-Weasel** [comment]
Oh TicTacToe nice, love it,  
i'll test it next days and give u a feedback after.

Testsystem #1 (W10 / x64 / ESET Smart Security Premium)
Testsystem #2 (W10 / x64 / only Windows defender)

**Comment by gucio321** [comment]
@GoWeasel how is it going?
